### ISSUE TITLE
Hi, I am very appreciated for your paper and codes, while arethere some codes lost in the file "DN_real"?

### ISSUE THREAD

**Issue by Jackyinuo** [jianzhangcs/DAGL#2]
Are there some codes lost in the file "DN_real"?
Compared with file "DN_Gray", "CAR" and “Demosaic”， "train.py" and the instruction about how to train and test DAGL are lost.
I have sincerely appreciated it if your beautiful codes become more perfect.

**Comment by MC-E** [comment]
Yes, the training code for **real noise** is not open source yet. We will update it soon.

**Comment by ZhengJianwei2** [comment]
请问作者real noise 的 training code有后续的公开计划吗

**Comment by Jiangjiawei2** [comment]
非常感谢作者无私分享代码以供学习，想请问下后续real noise的 training code会公开吗
